ui/k-info-panel/switch_info.kytos

```diff
@@ -184,6 +219,26 @@
        }
      }
    },
+   computed: {
+     get_options() {
+       var temp = []
```

Comment:
Could give it a more meaningful name for this variable?

Reply:
Changed on commit [a48b869d69b914523a9b9bd606c8cdc74059e42e](https://github.com/kytos-ng/topology/pull/81/commits/a48b869d69b914523a9b9bd606c8cdc74059e42e)